src/loader-xerces.cpp

```diff
@@ -196,6 +196,18 @@ SWIDStruct XercesSWIDTagIO::load(const string & filename) {
 }
 
 
+void XercesSWIDTagIO::save(const string & filename, const SWIDStruct & what) {
+	try {
+		XMLIO::save(filename, what);
+	} catch (const XMLException & toCatch) {
+		auto message = XMLString::transcode(toCatch.getMessage());
+		throw create_save_error(filename, message);
+	} catch (...) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "catch" a specific exception type. ([cpp:S2738](https://sonarcloud.io/organizations/openscap/rules#rule_key=cpp%3AS2738))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=openscap-libswid&pullRequest=9&issues=AWNj9DzYXOBV_YrhWXjj&open=AWNj9DzYXOBV_YrhWXjj)

[ISSUE_KEY]: AWNj9DzYXOBV_YrhWXjj